projects/cds/src/merchandising/facade/cds-merchandising-user-context.service.spec.ts

```diff
@@ -10,13 +10,15 @@ import {
 } from '@spartacus/core';
 import { Observable, of } from 'rxjs';
 import { map } from 'rxjs/operators';
+import { ProfileTagEventService } from './../../profiletag/services/profiletag-event.service';
 import {
   MERCHANDISING_FACET_NORMALIZER,
   MERCHANDISING_FACET_TO_QUERYPARAM_NORMALIZER,
 } from './../connectors/strategy/converters';
 import { MerchandisingUserContext } from './../model/merchandising-user-context.model';
 import { CdsMerchandisingUserContextService } from './cds-merchandising-user-context.service';
 
+const uuidv4 = require('uuid/v4');
```

Comment:
Can this also be changed to use a mocked UUID?

Reply:
Done

Reply:
I think you are still using the uuid/v4 to generate uuids

Reply:
Yup, missed that one, done it now